### PR TITLE
Point to correct resource

### PR DESCRIPTION
**Pull request by joinimran**
Updated link to point to correct source.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6352

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@mypil FYI Please

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Organizations using older directory synchronization technology, such as DirSync
...
+You can deploy Windows Hello for Business key trust in non-federated and federated environments.  For non-federated environments, key trust deployments work in environments that have deployed [Password Synchronization with Azure AD Connect](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/whatis-phs) or [Azure Active Directory Pass-through-Authentication](https://docs.microsoft.com/azure/active-directory/connect/active-directory-aadconnect-pass-through-authentication).  For federated environments, you can deploy Windows Hello for Business key trust using Active Directory Federation Services (AD FS) 2012 R2 or later.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]:
```suggestion
You can deploy Windows Hello for Business key trust in non-federated and federated environments.  For non-federated environments, key trust deployments work in environments that have deployed [Password Synchronization with Azure AD Connect](https://docs.microsoft.com/azure/active-directory/hybrid/whatis-phs) or [Azure Active Directory Pass-through-Authentication](https://docs.microsoft.com/azure/active-directory/connect/active-directory-aadconnect-pass-through-authentication).  For federated environments, you can deploy Windows Hello for Business key trust using Active Directory Federation Services (AD FS) 2012 R2 or later. 
```
I have verified that the language agnostic version of this link works, so we should skip the `/en-us/` part when possible.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Organizations using older directory synchronization technology, such as DirSync
 
 
 ## Federation with Azure ##
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md]: The pound signs ("hashtags") after this heading do not offer any MarkDown functionality and are therefore ignored (and redundant).
```suggestion
## Federation with Azure
```

**Comment by joinimran** [comment]
Thanks @illfated , Ready to go live @mypil

**Comment by mypil** [comment]
@mapalko - PR has been copy edited and is ready for final review. This is based on issue #6352.

Thank you.

cc: @kenwith

**Pull request merged**